lib/Backend/Inline.cpp

```diff
@@ -4677,8 +4677,7 @@ Inline::MapFormals(Func *inlinee,
             }
             if (instr->m_func != inlinee)
             {
-                // this can happen only when we are inlining a function which has inlined an apply call with the arguments object
-                formalCount = instr->m_func->GetJITFunctionBody()->GetInParamsCount();
```

Comment:
>formalCount [](start = 16, length = 11)

So the earlier fix about formalCount being wrong during NewScArray creation - is also taken care? - can you also pls  verify that?